src/components/CardDetails.js

```diff
@@ -1,16 +1,186 @@
+import {
+  Card,
+  Divider,
+  ListItem,
+  Button,
+  Icon,
+  SearchBar,
+  Text,
+} from 'react-native-elements';
+
+import ImageSliderz from 'react-native-image-slideshow';
 
 function CardDetails(props) {
-  const productDetails = useSelector(state => state)
-  console.log("cart", productDetails);
+  const productDetails = useSelector((state) => state.productDetails);
+  const [quantity, setQuantity] = useState(1);
+  const price = productDetails.price.slice(1);
+  const [total, setTotal] = useState(price);
+
+  console.log('cart', productDetails.picture);
   // take the data from productDetails
+
+  const onPressDec = () =>
+    setQuantity((prevCount) => (prevCount != 1 ? prevCount - 1 : 1));
+  const onPressInc = () => setQuantity((prevCount) => prevCount + 1);
+
+  useEffect(() => {
+    setTotal(price * quantity);
+  }, [quantity]);
+
   return (
-    <View >
-      <Text>hi</Text>
-    </View >
+    <ScrollView>
+      <View style={styles.container}>
+        <Card containerStyle={styles.card}>
+          <ImageSliderz
+            dataSource={[
+              {
+                url: productDetails.picture,
+              },
+              {
+                url: productDetails.picture,
+              },
+              {
+                url: productDetails.picture,
+              },
+            ]}
+            height={250}
+          />
+          <View style={styles.text}>
+            <Card.FeaturedTitle style={styles.cardName}>
+              <Icon name="price-tag" type="entypo" size={26} color="#015A20" />
+
+              {productDetails.name}
+            </Card.FeaturedTitle>
+
+            <Card.FeaturedTitle style={styles.cardPrice}>
+              {productDetails.price}
+            </Card.FeaturedTitle>
+          </View>
+        </Card>
+        <View style={styles.description}>
+          <Text h4 h4Style={{fontSize: 15, paddingBottom: 8}}>
+            Description :
+          </Text>
+          <Text>{productDetails.brief}</Text>
+        </View>
+
+        <View style={styles.amount}>
+          <View style={styles.amount}>
+            <TouchableOpacity onPress={onPressDec}>
+              <Icon
+                name="circle-with-minus"
+                type="entypo"
+                size={26}
+                color="#015A20"
+              />
+            </TouchableOpacity>
+
+            <Text h4> {quantity} </Text>
+            <TouchableOpacity onPress={onPressInc}>
+              <Icon
+                name="circle-with-plus"
+                type="entypo"
+                size={26}
+                color="#015A20"
+              />
+            </TouchableOpacity>
+          </View>
+          <View>
+            <Text h4>
+              Total:{' '}
+              <Text h4 style={styles.total}>
+                ${total}
+              </Text>
+            </Text>
+          </View>
+        </View>
+        <TouchableOpacity style={styles.buttonContainer}>
```

Comment:
add function to add the product to the cart, you can use redux to save the items